chainer/functions/math/erfcx.py

```diff
@@ -27,7 +27,7 @@ def forward_cpu(self, x):
                 _erfcx_cpu = special.erfcx
             except ImportError:
                 raise ImportError('SciPy is not available. Forward computation'
-                                  ' of erfcx can not be done.')
+                                  ' of erfcx cannot be done.')
```

Comment:
Could you add 'in CPU' to make it consistent with `erfinv` and `erfcinv`?

Reply:
Changed to be consistent with `erfinv` and `erfcinv`.